syft/pkg/cataloger/cpe_specificity.go

```diff
@@ -1,31 +1,46 @@
 package cataloger
 
-import "github.com/facebookincubator/nvdtools/wfn"
+import (
+	"sort"
+
+	"github.com/facebookincubator/nvdtools/wfn"
+)
+
+var _ sort.Interface = (*ByCPESpecificity)(nil)
 
 type ByCPESpecificity []wfn.Attributes
 
-// Implementing sort.Interface
 func (c ByCPESpecificity) Len() int      { return len(c) }
 func (c ByCPESpecificity) Swap(i, j int) { c[i], c[j] = c[j], c[i] }
 func (c ByCPESpecificity) Less(i, j int) bool {
-	return countSpecifiedFields(c[i]) > countSpecifiedFields(c[j])
+	iScore := countSpecifiedFields(c[i])
+	jScore := countSpecifiedFields(c[j])
+	if iScore == jScore {
```

Comment:
nit: Some vertical space surrounding the if block could help readers follow what's happening, and visualize the distinct concerns being addressed